cmd/services/graphService/main.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"github.com/diadata-org/diadata/pkg/dia"
+	"github.com/diadata-org/diadata/pkg/graph"
+	"github.com/diadata-org/diadata/pkg/model"
+	"log"
+	"os"
+	"time"
+)
+
+const (
+	GRAPH_PATH = "/charts/"
+)
+
+func main() {
+	log.SetFlags(log.LstdFlags | log.Lshortfile)
+
+	dataStore, err := models.NewDataStore()
+	if err != nil {
+		log.Println(err)
+		return
+	}
+
+	for {
+		for _, symbol := range dia.SymbolsFrontPage() {
+			points, err := dataStore.GetChartPoints(symbol)
+			if err != nil {
+				log.Println(err)
```

Comment:
do not simply print errors, it's a pain in the ass yo figure out where it comes from on a big codebase, use log.ErrorLn("blabla", err)